packages/next/README.md

```diff
@@ -1824,6 +1808,8 @@ export function render(req: http.IncomingMessage, res: http.ServerResponse) => v
 - [http.ServerResponse](https://nodejs.org/api/http.html#http_class_http_serverresponse)
 - `void` refers to the function not having a return value and is equivalent to JavaScript's `undefined`. Calling the function will finish the request.
 
+Using the serverless, you can deploy Next.js to [ZEIT Now](https://zeit.co/now) with all of the benefits and added ease of control like for example; [custom routes](https://zeit.co/guides/custom-next-js-server-to-routes/) and caching headers. See the [ZEIT Guide for Deploying Next.js with Now](https://zeit.co/guides/deploying-nextjs-with-now/) for more information.
```

Comment:
"Using the serverless" seems like this misses something

Reply:
Fixed!